app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -102,6 +137,23 @@ def set_quiz
         @quiz = Quiz.find(params[:quiz_id])
       end
 
+      def get_points(lobby, player)
+        questions = lobby.quiz.questions
+
+        questions.each do |question|
+          next unless received_points?(question, player)
+
+          question.points
+        end.sum
+      end
+
+      def received_points?(question, player)
+        correct_answers = question.answers.select(&:is_correct)
+        # rubocop:disable Metrics/LineLength
+        question unless (PlayerAnswer.where(player: player, answer: correct_answers).count == correct_answers.count) && (PlayerAnswer.where(player: player).count == correct_answers.count)
```

Comment:
This isn't really readable. You could use a normal `if` and break the conditions on a few new lines.

Reply:
It is also a method ending with `?` so the expected behaviour would be to return a `true` or `false` value. Here you're returning the `question`.